src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -36,6 +82,8 @@ const TopicFragment = ({
   searchText,
   withoutRecent = false
 }) => {
+  const [activeSections, setActiveSections] = React.useState([]);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of local state for `activeSections` is appropriate here as it's used for UI interactivity within this component only.
```diff
+ const [activeSections, setActiveSections] = React.useState([]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->